wiki/Article_Style_Guide/en.md

```diff
@@ -195,10 +235,14 @@ When refering to `osu!` (the game/framework itself, not the game mode), it shoul
 For example:
 
 - `The language that _osu!_ is written in is C# using the .NET Framework.`
+- `One of the game modes in _osu!_ is osu!catch.`
 
 ### Serial comma
 
-When listing items of 3 or more in a sentence, use the serial comma.
+_The serial comma is also known as the "Oxford comma" or the "Harvard comma"._
```

Comment:
This isn't supposed to be a _fun fact_, rather it is to say, "Oh, btw, if you didn't know already, `Serial comma` = `Oxford comma` = `Harvard comma`." So this is technically a notice, I guess.

It's supposed to be more clarifying because when I first heard about it, I was told that this is the "Oxford comma" (and have had never heard it as the "Serial Comma" or the "Harvard comma").